### ISSUE TITLE
Grants/revokes on functions

### ISSUE THREAD

**Issue by bfelbo** [djrobstep/migra#64]
Migra doesn't detect differences between databases related to granting access to functions. For instance: `GRANT execute ON FUNCTION my_schema.my_function to my_role;` There's also no tests ensuring this: https://github.com/djrobstep/migra/search?p=1&q=grant&unscoped_q=grant

**Comment by djrobstep** [comment]
Yes, privilege support isn't comprehensive atm. We'll need to add some functionality to `schemainspect` to handle this.

**Comment by bfelbo** [comment]
It would be extremely useful to have this :)

**Comment by tianhuil** [comment]
@djrobstep: I love Migra but need this feature.  Any chance we could add this?
